pkg/syncthing/api.go

```diff
@@ -30,8 +30,15 @@ type addAPIKeyTransport struct {
 	T http.RoundTripper
 }
 
+const (
+	APIKeyHeader      = "X-Api-Key"
+	APIKeyHeaderValue = "cnd"
+)
+
+// RoundTrip implements the http.RoundTripper interface and is used to add the
+// desired request headers to http requests.
```

Comment:
What's the context behind this change? Where does the value `cdn` come from?

Reply:
I've seen the same API KEY being set when we init syncthing, so it makes more sense now 👍

Reply:
fwiw I do not know the answer to your question on , "why we are sending this header?" The approach was to keep the code doing what it was doing before, but with a test.

I did the overall thing just to go through the flow of a unittest in okteto/okteto while investigating making changes for our approach to test coverage.